components/tabs/tabs.js

```diff
@@ -739,6 +792,43 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 		this.requestUpdate();
 	}
 
+	async _handleTabsSlotChange(e) {
+		this._defaultSlotBehavior = false;
+
+		this._tabs = e.target.assignedNodes({ flatten: true })
```

Comment:
Feel free to use [assignedElements](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSlotElement/assignedElements) instead. The API is the same as `assignedNodes` except that it only returns elements, making the the `node.nodeType === Node.ELEMENT_NODE` check unnecessary.

It took a while for all the browsers to support `assignedElements`, which is why most of our other code uses `assignedNodes`. But it is now safe for us to use.